### PR TITLE
fix: make both HDMI ports work on RPi 4

### LINKED ISSUE #3414: Make both HDMI ports work on Raspberry Pi 4
It seems that due to `hdmi_safe=1` in config.txt (see https://github.com/talos-systems/talos/pull/3115), only one HDMI port works, the other shows the rainbow. It should be easy to fix by settings that option for both ports. See https://www.raspberrypi.org/documentation/configuration/config-txt/video.md and https://www.raspberrypi.org/documentation/configuration/config-txt/conditional.md

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #3414.

Should be tested manually before merge.

- [x] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [x] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [x] you ran unit-tests (`make unit-tests`)

### CODE REVIEW

**Review by andrewrynhard — APPROVED** [review]

File: internal/app/machined/pkg/runtime/v1alpha1/board/rpi_4/rpi_4.go [hunk 1]
```diff
@@ -5,8 +5,8 @@
 package rpi4
 
 import (
+	_ "embed" //nolint:gci
```
Comment (AlekSi) [inline internal/app/machined/pkg/runtime/v1alpha1/board/rpi_4/rpi_4.go:8]: `//nolint:gci` is required by the combination of blank import, golang/lint issues (and abandonment) and gci being gci.

**Review by andrewrynhard — APPROVED** [review]
@AlekSi This is a simple enough change that you can force merge. I don't think you need to rebase.

**Comment by AlekSi** [comment]
That PR wasn't merged because I wanted to re-test it manually :) And yeah, it does not solve the issue.

**Pull request merged**